app/settings/surveys/attribute-editor.html

```diff
@@ -3,7 +3,7 @@
       <label translate="app.name">Name</label>
       <input type="text" ng-model="editAttribute.label">
    </div>
-   <div class="form-field" ng-if="editAttribute.input !== 'upload'">
+   <div class="form-field" ng-if="editAttribute.input !== 'upload' && editAttribute.type !== 'title' && editAttribute.type !== 'description'">
```

Comment:
This is kinda ugly.. could we move this to a `canSetDefault` method on the scope?